System/Console/ANSI/Types.hs

```diff
@@ -26,11 +30,24 @@ data Color = Black
            | White
            deriving (Eq, Ord, Bounded, Enum, Show, Read, Ix)
 
--- | ANSI colors come in two intensities
+-- | ANSI 8-bit "candy" colors come in two intensities
 data ColorIntensity = Dull
                     | Vivid
                     deriving (Eq, Ord, Bounded, Enum, Show, Read, Ix)
 
+
+-- | ANSI 8-bit "6 × 6 × 6 cube" rgb colors (216 colors). Each individual vomponent values is in range [0..5].
```

Comment:
typo: "vomponent". Also, should "values" be singular instead?

Reply:
Thanks, I will fix it!